$OpenBSD: patch-shell_Makefile_am,v 1.1 2013/03/29 18:18:43 ajacoutot Exp $

XXX our krb5 is tool old, so disable the user account panel for the time beeing

--- shell/Makefile.am.orig	Fri Mar 29 19:04:17 2013
+++ shell/Makefile.am	Fri Mar 29 19:04:27 2013
@@ -65,8 +65,7 @@ gnome_control_center_LDADD =						\
 	$(top_builddir)/panels/search/libsearch.la			\
 	$(top_builddir)/panels/sharing/libsharing.la			\
 	$(top_builddir)/panels/sound/libsound.la			\
-	$(top_builddir)/panels/universal-access/libuniversal-access.la	\
-	$(top_builddir)/panels/user-accounts/libuser-accounts.la
+	$(top_builddir)/panels/universal-access/libuniversal-access.la
 
 if BUILD_WACOM
 gnome_control_center_LDADD += $(top_builddir)/panels/wacom/libwacom-properties.la
